protocol/directory.go

```diff
@@ -39,18 +65,43 @@ func (d *ConiksDirectory) Update() {
 	}
 }
 
+// SetPolicies sets this ConiksDirectory's epoch deadline and VRF
+// private key.
 func (d *ConiksDirectory) SetPolicies(epDeadline merkletree.TimeStamp, vrfKey vrf.PrivateKey) {
 	d.policies = merkletree.NewPolicies(epDeadline, vrfKey)
 }
 
+// EpochDeadline returns this ConiksDirectory's current epoch deadline
+// as a timestamp.
 func (d *ConiksDirectory) EpochDeadline() merkletree.TimeStamp {
 	return d.pad.LatestSTR().Policies.EpochDeadline
 }
 
+// LatestSTR Returns this ConiksDirectory's latest STR.
```

Comment:
>  the system has "entered" epoch t and remains in epoch t until the next directory update

I think it should be `t+1`. Since it is *signed* tree root and in "current" epoch `t+1`, the tree root haven't been signed yet, "latest" makes more sense to me.